nixos/tests/freenet.nix

```diff
@@ -0,0 +1,20 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "freenet";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ nagy ];
+  };
+
+  nodes = {
+    machine = { ... }: {
+      services.freenet.enable = true;
+    };
+  };
+
+  testScript = ''
+    machine.start()
```

Comment:
machine is automatically started on first use, this is not needed.

Reply:
I have mostly seen this in other tests and just copied, but I will take it out.